src/startupscriptgenerator/src/common/outputHelper.go

```diff
@@ -23,27 +22,25 @@ func AppendScriptToExtractZippedOutput(scriptBuilder *strings.Builder, appDir st
 	}
 
 	if FileExists(fullZipFilePath) {
-		scriptBuilder.WriteString("\n")
-		scriptBuilder.WriteString("echo Extracting '" + consts.CompressedOutputFileName + "' contents...\n")
-		scriptBuilder.WriteString("cd \"" + intermediateDir + "\"\n")
-		scriptBuilder.WriteString("tar -xzf \"" + consts.CompressedOutputFileName + "\"\n")
-		scriptBuilder.WriteString("echo Done.\n")
+		scriptBuilder.AppendEmptyLine()
```

Comment:
What do you think of this API?

Reply:
I like the idea! would probably go for even shorter names (`builder.Chdir`, `builder.ExtractArchive`).
great initiative 👍